lib/rules/best-practises/no-empty-blocks.js

```diff
@@ -50,27 +75,37 @@ class NoEmptyBlocksChecker extends BaseChecker {
   }
 
   Block(node) {
-    // ignore empty constructor body when it's being defined to call a parent
-    // constructor
-    if (node.parent.isConstructor && node.parent.modifiers.length > 0) {
+    // Skip constructor with modifiers
+    if (node.parent.isConstructor && node.parent.modifiers && node.parent.modifiers.length > 0) {
       return
     }
 
-    const isFallbackFunctionBlock = isFallbackFunction(node.parent)
-    const isReceiveFunctionBlock = isReceiveFunction(node.parent)
-    if (isFallbackFunctionBlock || isReceiveFunctionBlock) {
-      // ignore empty blocks in fallback or receive functions
+    // Skip fallback and receive
+    if (isFallbackFunction(node.parent) || isReceiveFunction(node.parent)) {
       return
     }
-    this._validateChildrenCount(node, 'statements')
-  }
 
-  StructDefinition(node) {
-    this._validateChildrenCount(node, 'members')
-  }
+    if (node.parent.type === 'ModifierDefinition') {
+      if (this.allowEmptyModifiers) {
+        if (!this._hasUnderscore(node)) {
```

Comment:
I believe we can simply not handle modifiers without `_;` as they are forbidden by the compiler